README.md

```diff
@@ -1,2 +1,26 @@
 # HelloMicroverse
-Setup a Hello Microverse HTML file
+HelloMicroverse
+First Microverse project.
+
+Built With
+HTML and CSS
+
+Authors
+👤 Victor Gonzalez
+GitHub: @Vgonma
+Twitter: @Victor_GonMa
+LinkedIn: https://www.linkedin.com/in/victor-manuel-gonzalez-massimi-a77265124
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+@ceddddy
+@tarak-psdu
+
+📝 License
```

Comment:
- [ ] Please add a descriptive README file to your project by using [this template](https://github.com/microverseinc/readme-template) and customize it. Please make sure to keep only the main header section and do not keep them empty. Also add a license in order to make it more professional :+1: 
- [ ] Your project doesn't have a license. I would like to recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish and add the licence link to section in your readme dedicated for it in order to keep it more professional :+1: